test/cppsuite/test_harness/test.h

```diff
@@ -112,29 +113,30 @@ class test : public database_operation {
         int64_t cache_size_mb, duration_seconds;
         bool enable_logging, statistics_logging;
         configuration *statistics_config;
-        std::string statistics_type;
+        std::string debug_mode, statistics_type;
         /* Build the database creation config string. */
         std::string db_create_config = CONNECTION_CREATE;
 
         /* Get the cache size. */
         cache_size_mb = _config->get_int(CACHE_SIZE_MB);
+        db_create_config += ",cache_size=" + std::to_string(cache_size_mb) + "MB";
 
         /* Get the statistics configuration for this run. */
         statistics_config = _config->get_subconfig(STATISTICS_CONFIG);
         statistics_type = statistics_config->get_string(TYPE);
         statistics_logging = statistics_config->get_bool(ENABLE_LOGGING);
-
+        db_create_config += statistics_logging ? "," + std::string(STATISTICS_LOG) : "";
+        db_create_config += ",statistics=(" + statistics_type + ")";
         /* Don't forget to delete. */
         delete statistics_config;
 
-        db_create_config += ",statistics=(" + statistics_type + ")";
-        db_create_config += statistics_logging ? "," + std::string(STATISTICS_LOG) : "";
-        db_create_config += ",cache_size=" + std::to_string(cache_size_mb) + "MB";
-
         /* Enable or disable write ahead logging. */
         enable_logging = _config->get_bool(ENABLE_LOGGING);
         db_create_config += ",log=(enabled=" + std::string(enable_logging ? "true" : "false") + ")";
 
+        /* Add the user supplied wiredtiger open config. */
+        db_create_config += _wt_open_config;
```

Comment:
You can directly use the input `wt_open_config`

Reply:
This isn't in the constructor?

Reply:
Sorry, my new settings tricked me, I was not looking correctly.